hackathon_site/registration/models.py

```diff
@@ -99,9 +105,30 @@ class Application(models.Model):
         ],
         null=False,
     )
-    q1 = models.TextField(null=False, help_text="First question?", max_length=1000)
-    q2 = models.TextField(null=False, help_text="Second question?", max_length=1000)
-    q3 = models.TextField(null=False, help_text="Third question?", max_length=1000)
+    linkedin = models.CharField(
+        max_length=255, help_text="LinkedIn link", null=True, blank=True
+    )
+    github = models.CharField(
+        max_length=255, help_text="Github Link", null=True, blank=True
+    )
+    devpost = models.CharField(
+        max_length=255, help_text="Devpost Link", null=True, blank=True
+    )
+    q1 = models.TextField(
+        null=False,
+        help_text="Why do you want to participate in NewHacks?",
+        max_length=1000,
+    )
+    q2 = models.TextField(
+        null=False,
+        help_text="What is your technical experience with software?",
+        max_length=1000,
+    )
+    q3 = models.TextField(
+        null=False,
+        help_text="Describe any past experience with if any.",
+        max_length=1000,
+    )
     conduct_agree = models.BooleanField(
```

Comment:
for the checkboxes here we need to confirm with logistics what they want, last year we had 4 checkboxes:
- code of conduct aggreement (required)
- MLH aggreement (required)
- MLH email list signup (optional)
- sharing resume to sponsors (optional)
You can just copy those over from the 2021 repo